### DISCUSSION TITLE
Plans For This Project?

### DISCUSSION THREAD

**Discussion by ghost** [resteasy/resteasy-guice#11] [Q&A] [answered]
We have several services built using Java 8, RestEasy & Guice. We've been planning our upgrade of all three across our projects.

However the removal of GuiceResteasyBootstrapServletContextListener from RestEasy 5.x adds some complexity to the upgrades.

Are there concrete plans to maintain this port of the Guice integration and/or a roadmap to having a Final version released?

**Comment by jamezp** [comment, marked as answer]
@iain-davis The hope is to get some community support with this. All the projects I personally work with do not use Guice. However, I'm happy to do releases and merge pull requests.

**Comment by ghost** [comment]
Thank you for the reply!

That's completely understandable. It's a reasonable to need community support for specific combinations of technologies. :)

I may put the idea in front of teammates and see if any of us have the bandwidth available for a community project. I'm tempted, but I'm not confident I'll have the time.
